library/src/main/java/org/apache/apex/malhar/lib/window/Tuple.java

```diff
@@ -45,11 +45,12 @@
    */
   class PlainTuple<T> implements Tuple<T>
   {
-    private T value;
```

Comment:
@siyuanh you have a point. @tweise @vrozov do you guys know of any value in making tuples immutable?